<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Othello</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Game Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_H_VS_H">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Human VS Human</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_H_VS_P">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Human VS PC</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_P_VS_P">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PC VS PC</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_black">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Black</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_white">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>White</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pushButton_START">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
